docs/pages/lighting-setups.rst

```diff
@@ -212,3 +212,17 @@ An existing object's light setup can be modified at any time with :ref:`Simulato
 .. code:: python
 
     sim.set_object_light_setup(my_object_id, "my_custom_lighting_key")
+
```

Comment:
I didn't flesh out this RST content with new screenshots or otherwise because this is all throwaway. We're going to replace this RST tutorial with a Colab-based tutorial soon.